stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/DtoMapper.kt

```diff
@@ -0,0 +1,59 @@
+package io.getstream.chat.android.client.api2
+
+import io.getstream.chat.android.client.api2.model.dto.DownstreamMessageDto
+import io.getstream.chat.android.client.api2.model.dto.UpstreamMessageDto
+import io.getstream.chat.android.client.api2.model.dto.UserDto
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.User
+
+internal class DtoMapper {
```

Comment:
Might want to split this out into multiple classes or rename it later, but probably good enough for now

Reply:
Probably gonna go with splitting out into extensions https://github.com/GetStream/stream-chat-android/pull/1274#discussion_r565174986